.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: java
 
 jdk:
-  - oraclejdk7
```

Comment:
One possibility in case you still want to test on JDK7 would be to use a Maven profile for errorprone that's only activated when JDK==8 (errorprone won't work with JDK 9 either)